packages/blockchain-wallet-v4-frontend/src/graphql/queries/NftAssets.graphql

```diff
@@ -6,6 +6,9 @@ query Assets($filter: [AssetFilter], $traitFilter: [TraitFilter], $sort: AssetSo
     contract {
       address
     }
+    owners {
+      address
+    }
```

Comment:
do you need this? you should already be able to tell if user is owner if the asset is on the users owned asset page and they are authenticated.